src/test/java/io/xpring/payid/PayIDUtilsTest.java

```diff
@@ -41,11 +39,11 @@ public void testParsePaymentPointerWithWellKnownPathAndTrailingSlash() {
     String rawPaymentPointer = "$example.com";
 
     // WHEN it is parsed to a PaymentPointer object
-    PaymentPointer paymentPointer = PayIDUtils.parsePayID(rawPaymentPointer);
+    PayIDComponents paymentPointer = PayIDUtils.parsePayID(rawPaymentPointer);
 
     // THEN the host and path are set correctly.
     assertEquals(paymentPointer.host(), "example.com");
-    assertEquals(paymentPointer.path(), PaymentPointer.WELL_KNOWN);
+    assertEquals(paymentPointer.path(), "/.well-known/pay");
```

Comment:
So the new payment pointer format comes later?

Reply:
Correct, #198